### PR TITLE
Clean up GPU support page

### PR DESCRIPTION
**Pull request by ru-fu**
Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

File: exp/gpu-support.md [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
+<a name="supported-gpus"></a>
 ## Supported GPUs
 
 Anbox Clouds allows access to GPUs from Intel, AMD and Nvidia inside the Anbox container. Concrete support for the individual GPU depends on the platform being using for Anbox. The included `webrtc` platform currently supports the following GPUs:
 
-| Vendor | Model          | Render | Hardware Video Encode |
+| Vendor | Model          | Render | Hardware video encode |
 |--------|----------------|--------|-----------------------|
 | AMD    | WX5100, WX4100 | Yes    | No                    |
 | Nvidia | Quadro, Tesla  | Yes    | Yes                   |
```
Comment (morphis) [inline exp/gpu-support.md:13]: While we're on it we need to clean this up. For NVIDIA we should say `Quadro, Tesla, Ampere`

File: exp/gpu-support.md [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
 
 See [GPU slots](https://discourse.ubuntu.com/t/about-capacity-planning/28717#gpu-slots) for detailed information.
 
-## Using GPUs inside a Container
+## Using GPUs inside a container
 
-AMS configures a LXD container to pass through a GPU device from the host. As of right now all GPUs available to a machine are passed to every container owning a GPU slot. For Nvidia GPUs LXD uses the [Nvidia container runtime](https://github.com/NVIDIA/nvidia-container-runtime) to make the GPU driver of the host available to a container. When GPUs from Intel or AMD are being used no GPU driver is made available automatically. It has to be provided by an [addon](https://discourse.ubuntu.com/t/managing-addons/17759).
+AMS configures each LXD container to pass through a GPU device from the host. As of right now, all GPUs that are available to a machine are passed to every container that owns a GPU slot. For Nvidia GPUs, LXD uses the [Nvidia container runtime](https://github.com/NVIDIA/nvidia-container-runtime) to make the GPU driver of the host available to the container.
```
Comment (morphis) [inline exp/gpu-support.md:33]:
```suggestion
AMS configures each LXD container to pass through a GPU device from the host. As of right now, all GPUs that are available to a machine are passed to every container that owns a GPU slot. For NVIDIA GPUs, LXD uses the [NVIDIA container runtime](https://github.com/NVIDIA/nvidia-container-runtime) to make the GPU driver of the host available to the container.
```

File: exp/gpu-support.md (hunk 1 above)
Comment (morphis) [inline exp/gpu-support.md:13]:
```suggestion
| NVIDIA | Quadro, Tesla  | Yes    | Yes                   |
```

File: exp/gpu-support.md [hunk 3]
```diff
@@ -1,40 +1,45 @@
-Anbox Cloud has support to manage GPUs and can provide them to individual containers for rendering and video encoding functionality.
+Anbox Cloud has support for managing GPUs and can provide them to individual containers for rendering and video encoding functionality.
 
-In case that no GPU is available Anbox Cloud automatically falls back to software rendering and video encoding. This makes it possible to run entirely without a GPU.
+If no GPU is available, Anbox Cloud automatically falls back to software rendering and video encoding. This makes it possible to run entirely without a GPU.
 
+<a name="supported-gpus"></a>
 ## Supported GPUs
 
 Anbox Clouds allows access to GPUs from Intel, AMD and Nvidia inside the Anbox container. Concrete support for the individual GPU depends on the platform being using for Anbox. The included `webrtc` platform currently supports the following GPUs:
```
Comment (morphis) [inline exp/gpu-support.md:8]:
```suggestion
Anbox Clouds allows access to GPUs from Intel, AMD and NVIDIA inside the Anbox container. Concrete support for the individual GPU depends on the platform being using for Anbox. The included `webrtc` platform currently supports the following GPUs:
```

**Review by morphis — CHANGES_REQUESTED** [review]

File: exp/gpu-support.md [hunk 4]
```diff
@@ -1,40 +1,45 @@
-Anbox Cloud has support to manage GPUs and can provide them to individual containers for rendering and video encoding functionality.
+Anbox Cloud has support for managing GPUs and can provide them to individual containers for rendering and video encoding functionality.
 
-In case that no GPU is available Anbox Cloud automatically falls back to software rendering and video encoding. This makes it possible to run entirely without a GPU.
+If no GPU is available, Anbox Cloud automatically falls back to software rendering and video encoding. This makes it possible to run entirely without a GPU.
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply exp/gpu-support.md]: Yes - that's the `--enable-graphics` flag, correct?
Tried to clarify it. :)

File: exp/gpu-support.md (hunk 2 above)
Comment (ru-fu) [reply exp/gpu-support.md]: Fixed globally (in a separate commit).

File: exp/gpu-support.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (morphis) [reply exp/gpu-support.md]: Yes, it is.

File: exp/gpu-support.md (hunk 2 above)
Comment (morphis) [reply exp/gpu-support.md]: Thanks!

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**